packages/webawesome/docs/_includes/sidebar.njk

```diff
@@ -399,13 +399,34 @@
 
 <wa-divider style="--spacing: var(--wa-space-xl);"></wa-divider>
 
-<div class="wa-stack wa-gap-xs" id="colophon">
-  {% include "logo-simple.njk" %}
-  <div class="wa-cluster wa-gap-xs">
-    <h2 class="wa-heading-xs">Web Awesome</h2>
-    <wa-badge id="version-beta-badge" variant="orange" appearance="filled" style="font-size: var(--wa-font-size-2xs);">Beta</wa-badge>
-    <wa-tooltip for="version-beta-badge" distance="2" style="font-size: var(--wa-font-size-xs);">Here be freshly made Awesome&hellip; and possible dragons</wa-tooltip>
+<div class="wa-stack wa-gap-xl" id="colophon">
+  <div class="wa-stack wa-gap-xs">
+    {% include "logo-simple.njk" %}
+    <div class="wa-cluster wa-gap-xs">
+      <h2 class="wa-heading-xs">Web Awesome</h2>
+      <wa-badge id="version-beta-badge" variant="orange" appearance="filled" style="font-size: var(--wa-font-size-2xs);">Beta</wa-badge>
+      <wa-tooltip for="version-beta-badge" distance="2" style="font-size: var(--wa-font-size-xs);">Here be freshly made Awesome&hellip; and possible dragons</wa-tooltip>
+    </div>
+    <p class="wa-caption-s">Version {{ package.version }}</p>
+    <p class="wa-caption-s">&copy; Fonticons, Inc.</p>
+  </div>
+
+  <div class="wa-cluster wa-caption-s the-socials">
```

Comment:
<img width="370" height="104" alt="CleanShot 2025-10-01 at 11 41 48@2x" src="https://github.com/user-attachments/assets/d39f06cb-b9c9-49e2-bbd9-afbe2474d804" />

The hit area is kinda small. Should we keep the same spacing but pad them out?